Stylized Races document

bleehu opened the pull request:
To facilitate HTML display of rules in CXDocs and make reading easier.

bleehu committed:
change races to have the agreed upon style of headers

bleehu committed:
Merge branch 'dev' into MH-Stylize-Races

bleehu committed:
Stylized Races and added some headings

1sourcecontrol commented on CharacterCreation/01_Races.txt:
```diff
@@ -1,57 +1,72 @@
-blend in with the environment, whether it be large tree cities or hanging 
-cliff cities.
-They typically believe in spiritualism, and their level of science has been 
+===== Society: =====
```
nix colon plox :)

bleehu replied:
wilco

1sourcecontrol commented on CharacterCreation/01_Races.txt:
```diff
@@ -72,13 +87,21 @@ requires dim light to see well.
+
+===== Society ===== 
+    Slave soldiers. Squad families.
+    Security Drones are specialized drones that have been granted a greater AI 
```
I think these two lines should be combined. 4 words, 2 sentences, and one 1 paragraph doesn't look great.

bleehu replied:
wilco.

1sourcecontrol reviewed (changes requested):
I'm not a fan of large quantity of blank lines, especially between a section declaration + paragraph (1) and the new lines between paragraphs (2). 

1.
![image](https://user-images.githubusercontent.com/2192156/33397140-905358d8-d4ff-11e7-9a02-fcf0b565c387.png)

2.
![image](https://user-images.githubusercontent.com/2192156/33397145-97bb3852-d4ff-11e7-8dff-acaf7c08bb09.png)

It will be harder for readers to scroll through the doc on laptops or smaller devices to get to the information they need.

bleehu committed:
Changed backtick character to ascii-compliant apostrophe

bleehu committed:
one more backtick

1sourcecontrol commented on CharacterCreation/01_Races.txt:
```diff
@@ -120,87 +149,131 @@ Starting Stat Points: 40
+    Your fingers hide a basic toolkit, including low-speed motorized 
+screwdrivers, soldering iron, wrenches, quantum spanner, toothpick, pliers, 
+sharp edge, lock pick, etc. If it fits in a leatherman, its plausible that it 
+would be in this toolkit.
```
Aren't "Integrated Multitool" and "Implanted Weapon" cybernetic augmentations? This listing makes it look like cyborgs come with them.

bleehu replied:
They are. Please note how the listing for them say to "see below" Where this is the "below". Side note; I really don't like how we reference the Leatherman brand. But that would be scope creep.

1sourcecontrol replied:
Ah, I see them in the list now.

I still think it will cause confusion, they way you set them out from all the others with sub-headings. 
Is this too hard on the parser?:
![image](https://user-images.githubusercontent.com/2192156/33746205-9be209de-db70-11e7-870e-9753d148e63f.png)

Yeah, Leatherman probably should be removed. Speaking of scope creep, this doc needs a spell check.

bleehu replied:
There's two reasons not to include the whole segment in the list. The first is that the parser doesn't track indents yet. But we can fix that, it would just take time and a rearchitecture. 
But the second reason is that it won't display well. The typography of having a list with 6 small items and two other items in the list which are much larger is all wrong. It won't scan well. Even in the best display case, the CSS will move things weird places when displayed on devices with different size screens.

1sourcecontrol commented on CharacterCreation/01_Races.txt:
```diff
@@ -120,87 +149,131 @@ Starting Stat Points: 40
-Tetra
+== Tetra ==
+
 -1 to DEX, -1 to STR
```
The copyright notice at the bottom needs to get removed

bleehu replied:
removed

1sourcecontrol commented on CharacterCreation/01_Races.txt:
```diff
@@ -72,13 +87,21 @@ requires dim light to see well.
+==============================
+
+===== Society ===== 
+    Slave soldiers. Squad families.
```
Missing a space between "Society" and the following line; for consistency.

bleehu replied:
added

bleehu committed:
Spell check and copywrite removal.

bleehu committed:
added space for consistency

bleehu committed:
Merge branch 'dev' into MH-Stylize-Races

also cleaned up racial traits

 Conflicts:
	CharacterCreation/01_Races.txt

Turtlelord26 commented on CharacterCreation/01_Races.txt:
```diff
@@ -1,49 +1,62 @@
 Speed:		Normal
 Starting Stat Points: 42
 Stat Modifiers:	
 +2 stat points to spend at level 1
```
Question: Will lines 16-17 properly parse into the stat block, or is it going to see line 16 as a topic heading?

bleehu replied:
It would see line 16 as a topic heading. Fixed in subsequent pull.

Turtlelord26 commented on CharacterCreation/01_Races.txt:
```diff
@@ -1,49 +1,62 @@
 Tundari
+==============================
+
 Society: Tribal system, lead by a council of elders.
```
Blocker - Trash. Line 28 is redundant with line 48-50 and non-compliant with style.

bleehu replied:
Took out the trash.

Turtlelord26 replied:
I still see it in the Files Changed view.

Turtlelord26 commented on CharacterCreation/01_Races.txt:
```diff
@@ -90,21 +121,27 @@ Starting Stat Points: 40
 
-//============================================================
+
+
```
Nonblocker: Excess blank lines at 136-137

bleehu replied:
trimmed excess blank lines

Turtlelord26 commented on CharacterCreation/01_Races.txt:
```diff
@@ -116,45 +153,73 @@ Starting Stat Points: 40
-Pick any three augments (you may take a single augment no more than 2 times, 
+===== Racial Traits =====
+
+    Pick any three augments (you may take a single augment no more than 2 times, 
```
Nonblocker: I don't like that the other two cyborg traits get Subsection headers but this one does not. It would be consistent to have a == Cybernetic Augmentation == subsection header between 156 and 158.

Turtlelord26 commented on CharacterCreation/01_Races.txt:
```diff
@@ -116,45 +153,73 @@ Starting Stat Points: 40
 Opaleite
+==============================
+
+    Opaleites are so named by the humans who encountered them for the way the 
```
Nonblocker: This section is unique in the document for not being attached to a section header. This is an inconsistency. Based on the paragraph in question, I conjecture it would appreciate an ===== Anatomy ===== header between 205 and 207.

Turtlelord26 commented on CharacterCreation/01_Races.txt:
```diff
@@ -165,68 +230,92 @@ can function both underwater and above water. When in the water, the lungs merel
+IV. A few stubborn stragglers remain on Trioch II, which is slowly becoming a 
+sandy, dry wasteland with very little atmosphere. 
+
+
```
Nonblocker: excess blank line at 246.

Turtlelord26 reviewed (changes requested):
Blocking Inconsistency: Tundari racial traits are formatted as Topics with paragraphs, while Security Drone traits are formatted as a stat block, Cyborgs use Subsections, and Opaleites have a hybridized list-and-paragraph. We should at least talk about standardizing.

The Opaleite subrace section also suffers from a lot of syntactic inconsistency. In particular, I'm unsure of the stylistic compliance of lines 265-267, 291-292, 303-304.

bleehu committed:
Merge branch 'dev' into MH-Stylize-Races

Turtlelord26 committed:
Addressing Racial Stylization Review

Handled all blocking and nonblocking issues from my earlier review, except for the inconsistent handling of racial trait style. I didn't want to unilaterally pick a way to do that, as I'm not on the team responsible for generating the traits.

bleehu committed:
took out the trash in races doc

Turtlelord26 committed:
Merge pull request #282 from bleehu/Turtlelord26-Stylize-Races_Review

Addressing Racial Stylization Review

bleehu commented:
The production instance of cx docs should pick this up automatically in a couple of minutes.
